package.json

```diff
@@ -16,20 +16,25 @@
   },
   "source": "src/index.ts",
   "exports": {
-    "require": "./dist/cow-sdk.js",
-    "default": "./dist/cow-sdk.modern.js"
+    "require": "./dist/src/cow-sdk.js",
+    "default": "./dist/src/cow-sdk.modern.js"
   },
-  "main": "./dist/cow-sdk.js",
-  "module": "./dist/cow-sdk.module.js",
-  "unpkg": "./dist/cow-sdk.umd.js",
+  "main": "./dist/src/cow-sdk.js",
+  "module": "./dist/src/cow-sdk.module.js",
+  "unpkg": "./dist/src/cow-sdk.umd.js",
   "scripts": {
     "build": "microbundle -f modern,esm,cjs",
     "start": "microbundle -f modern,esm,cjs watch",
     "lint": "eslint src",
     "format": "prettier --write \"src/**/*.+(ts|json)\"",
-    "test": "jest"
+    "test": "jest",
+    "prepare": "npm run build",
+    "prepublishOnly": "npm test && npm run lint",
+    "preversion": "npm run lint",
+    "version": "npm run format && git add -A src",
```

Comment:
These commands are a bit confusing to me.
What would be the workflow where you'd use each?

Reply:
they all run automatically when you publish a anew version

I copied the from somewhere, the most important above is the `prepare`, cause we were not building when publishing. The other are nice to have. 

* `prepare`: Runs any time before the package is packed, i.e. during npm publish and npm pack
* `prepublishOnly`: Runs BEFORE the package is prepared and packed, ONLY on npm publish.

The other doesn't harm, I'm fine with removing them. Although they make it easier to generate a version, cause it pushes the tags already. I got it here if it helps https://itnext.io/step-by-step-building-and-publishing-an-npm-typescript-package-44fe7164964c

Reply:
Ah ok, they "magically" plug into something else.
Not against having them, just trying to understand how/where they were used